package/cifs/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:16.04
+RUN apt-get update && \
+    apt-get install -y jq curl nfs-common
```

Comment:
hi @LLParse ,
I added `cifs-utils` to my Dockerfile based on Microsoft instructions:
https://docs.microsoft.com/en-us/azure/storage/storage-how-to-use-files-linux
This image been in use on some of our prod environments for couple months now without any hiccups.
let me know if you still wanna replace that package.